### PR TITLE
Update zero-trust-identity-device-access-policies-sharepoint.md

### PR DESCRIPTION
**Pull request by rafalfitt**
If SharePoint permissions are granted, users can edit files in their web browser.

"Users on unmanaged devices will have browser-only access with no ability to download, print, or sync files. They also won't be able to access content through apps, including the Microsoft Office desktop apps. When you limit access, you can choose to allow or block editing files in the browser."

https://learn.microsoft.com/en-us/sharepoint/control-access-from-unmanaged-devices#:~:text=no%20ability%20to%20download%2C%20print%2C%20or%20sync%20files

### CODE REVIEW

**Comment by ShannonLeavitt** [comment]
@chrisda

Can you review the proposed changes? 

IMPORTANT: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

**Review by chrisda — CHANGES_REQUESTED** [review]
Add a comma after 'printing'. I can't because I don't have Write access in this repo.

**Comment by chrisda** [comment]
#sign-off

File: security-docs/zero-trust/zero-trust-identity-device-access-policies-sharepoint.md [hunk 1]
```diff
@@ -67,7 +67,7 @@ To configure this policy see "Block or limit access to specific SharePoint site
 
 Microsoft recommends you protect content in SharePoint sites with enterprise and specialized security content with device access controls. You do this by creating a policy that specifies the level of protection and the sites to apply the protection to.
 
-- Enterprise sites: Allow browser-only access. This prevents users from editing and downloading files.
+- Enterprise sites: Allow browser-only access. This prevents users from downloading, printing or syncing files.
```
Comment (Jak-MS) [inline security-docs/zero-trust/zero-trust-identity-device-access-policies-sharepoint.md]:
```suggestion
- Enterprise sites: Allow browser-only access. This prevents users from downloading, printing, or syncing files.
```

**Pull request merged**